### PR TITLE
Updates events url to include order_by

### PR DESCRIPTION
**Pull request by pfarnach**
Note that, as per my convo with @InciteDemocracy, the intended functionality is for a selected date to be a "start date", meaning it will display events for all days in the future

### CODE REVIEW

File: src/api/eventsAPI.js [hunk 1]
```diff
@@ -5,9 +5,12 @@ import { queryBuilder } from '../utils';
 const BASE_URL = 'https://resistance-calendar.herokuapp.com/v1/events';
 
 function getEvents({ page = 0, perPage = 25 }, filterParams) {
+  const pageUrl = `page=${page}`;
+  const perPageUrl = `per_page=${perPage}`;
+  const orderByUrl = '$orderby=start_date desc';
```
Comment (pdw207) [inline src/api/eventsAPI.js]: Should we have test coverage for order by?

**Comment by pdw207** [comment]
Nice!

File: src/api/eventsAPI.js (hunk 1 above)
Comment (pfarnach) [inline src/api/eventsAPI.js]: Hmm I'd love to, but how would I do that?  It seems like that's in the realm of the server right?

Comment (pdw207) [inline src/api/eventsAPI.js]: Sorry, I was thinking that we would also have a default sort and UI where users can sort results.

**Pull request merged**